.github/workflows/release_bin.yaml

```diff
@@ -23,4 +24,4 @@ jobs:
           version: latest
           args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
is this an oversight? not sure if this is required?

Reply:
oopsie yes, my bad